### PR TITLE
3.3.0 release

### PR DESCRIPTION
**Pull request by BinamB**
bump up version in pyproject.toml for release.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,18 +8,19 @@ readme = "README.md"
 repository = "https://github.com/uc-cdis/dictionaryutils" 
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "~3.5 || ~3.6"
 PyYAML = "~=5.1"
-jsonschema = ">=2.5,<3.0"
+jsonschema = ">2.5,<4.0"
```
Comment (paulineribeyre) [inline pyproject.toml:13]: is the change from `>=` to `>` necessary?

File: pyproject.toml [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
 requests = "~=2.18"
 
 [tool.poetry.dev-dependencies]
 nose = ">=1.3.7,<2.0.0"
 cdislogging = "^1.0.0"
-data-simulator = {git = "https://github.com/uc-cdis/data-simulator.git", tag = "1.2.0"}
 pytest = "^5.4.3"
-gen3datamodel = "3.0.0"
+gen3datamodel = "3.0.2"
+jsonschema = ">=2.5,<4.0"
```
Comment (paulineribeyre) [inline pyproject.toml:22]: `jsonschema` is duplicated

File: pyproject.toml (hunk 1 above)
Comment (BinamB) [reply pyproject.toml:13]: oh thats not necessary but the `4.0` is.

File: pyproject.toml (hunk 2 above)
Comment (BinamB) [reply pyproject.toml:22]: removing

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**